File: 1935-maximum-number-of-words-you-can-type/1935-maximum-number-of-words-you-can-type.java
class Solution {
    public int canBeTypedWords(String text, String brokenLetters) {
        int result=0;
        
        Set<Character> set= new HashSet<>();
        for(char c: brokenLetters.toCharArray())
        {
            set.add(c);
        }
        
        String[] arr=text.split(" ");
        
        for(int i=0; i<arr.length; i++)
        {
            for(int j=0; j<arr[i].length(); j++)
            {
                if(set.contains(arr[i].charAt(j)))
                {
                    result--;
                    break;
                }
            }
            result++;
        }
        return result;
    }
}

File: 1935-maximum-number-of-words-you-can-type/1935-maximum-number-of-words-you-can-type.cpp
class Solution {
public:
    int canBeTypedWords(string text, string brokenLetters) {
        
        unordered_set<char> set;
        int count=0;
        int i=0;
        bool t=false;
        
        for(int i=0; i<brokenLetters.size();i++)
        {
            set.insert(brokenLetters[i]);
        }
        
        while(i<=text.size())
        {
            if(set.count(text[i]))
            {
                t=true;
            }
            else if(text[i]==' ' || text[i]=='\0')
            {
                if(!t)
                {
                    count++;
                }
                t=false;
            }
            i++;
        }
        return count;
    }
};